front/app/containers/IdeasNewPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -156,7 +159,7 @@ const IdeasNewSurveyForm = ({ project }: Props) => {
     const requestBody = {
       ...data,
       project_id: project.data.id,
-      // phase_ids: [phaseId], // TODO: JS - should only be added if moderator
+      phase_ids: userIsModerator ? [phaseId] : [],
```

Comment:
Could you still add the comment as Sebi did [here](https://github.com/CitizenLabDotCo/citizenlab/pull/7291/files)? It's not obvious from this code why we only sent it when user is moderator, so let's document. :)

Reply:
Yep, just have